Mathlib/Analysis/Distribution/SchwartzSpace.lean

```diff
@@ -21,8 +21,8 @@ public import Mathlib.MeasureTheory.Function.L2Space
 # Schwartz space
 
 This file defines the Schwartz space. Usually, the Schwartz space is defined as the set of smooth
-functions `f : ℝ^n → ℂ` such that there exists `C_{αβ} > 0` with `|x^α ∂^β f(x)| < C_{αβ}` for
-all `x ∈ ℝ^n` and for all multiindices `α`, `β`.
+functions $f : ℝ^n → ℂ$ such that there exists $C_{αβ} > 0$ with $$|x^α ∂^β f(x)| < C_{αβ}$$ for
```

Comment:
```suggestion
functions $f : ℝ^n → ℂ$ such that there exists $C_{αβ} > 0$ with $|x^α ∂^β f(x)| < C_{αβ}$ for
```
Or was this intentionally meant to be a displaymath formula?

Feel free to reject this one.

Reply:
I really don't have strong opinions on this, single dollars are fine with me